plugins/modules/ndo_interface_setting.py

```diff
@@ -0,0 +1,808 @@
+    type: int
+  link_level_bring_up_delay:
+    description:
+    - The time in milliseconds that the decision feedback equalizer (DFE) tuning is delayed when a port is coming up.
+    - The default value is 0.
+    - The value must be an integer between 0 and 10000.
+    type: int
+  link_level_fec:
+    description:
+    - The Forwarding Error Correction (FEC) is used for obtaining error control in data transmission.
+    - The default value is C(inherit).
+    type: str
+    choices: [ inherit, cl74_fc_fec, cl91_rs_fec, cons16_rs_fec, ieee_rs_fec, kp_fec, disable_fec ]
+  l2_interface_qin_q:
+    description:
+    - The QinQ enables  mapping double-tagged VLAN traffic.
+    - The default value is C(disabled).
+    type: str
+    choices: [ core_port, double_q_tag_port, edge_port, disabled ]
+  l2_interface_reflective_relay:
+    description:
+    - The reflective relay enabled forwarding trafic.
+    - The default value is C(disabled).
+    type: str
+    choices: [ enabled, disabled ]
+  lldp_transmit_state:
+    description:
+    - The LLDP transmit state allows Link Layer Discovery Protocol (LLDP) packets to be sent from the interface.
+    - The default value is C(enabled).
+    type: str
+    choices: [ enabled, disabled ]
+  lldp_receive_state:
+    description:
+    - The LLDP receive state allows LLDP packets to be received by the interface.
+    - The default value is C(enabled).
+    type: str
+    choices: [ enabled, disabled ]
+  stp_bpdu_filter:
+    description:
+    - The STP Bridge Protocol Data Unit (BPDU) filter filters out any BPDUs on the port.
+    - The default value is C(disabled).
+    type: str
+    choices: [ enabled, disabled ]
+  stp_bpdu_guard:
+    description:
+    - The STP BPDU guard prevents the port from receiving BPDUs.
+    - The default value is C(disabled).
+    type: str
+    choices: [ enabled, disabled ]
+  llfc_transmit_state:
+    description:
+    - The LLFC transmit state alows Link Level Flow Control (LLFC) packets to be sent from the interface.
+    - The default value is C(disabled).
+    type: str
+    choices: [ enabled, disabled ]
+  llfc_receive_state:
+    description:
+    - The LLFC receive state allows LLFC packets to be received by the interface.
+    - The default value is C(disabled).
+    type: str
+    choices: [ enabled, disabled ]
+  mcp_admin_state:
+    description:
+    - The MCP admin state enables MisCabling Protocol (MCP) on the interface.
+    - The default value is C(enabled).
+    type: str
+    choices: [ enabled, disabled ]
+  mcp_strict_mode:
+    description:
+    - The MCP strict mode.
+    - The default value is C(off).
+    type: str
+    choices: [ 'on', 'off' ]
+  mcp_initial_delay_time:
+    description:
+    - The MCP initial delay time in seconds.
+    - The default value is 180.
+    - The value must be between 0 and 1800.
+    - The value is available only when the mcp_strict_mode is C(on).
+    type: int
+  mcp_transmission_frequency_sec:
```

Comment:
```suggestion
  mcp_transmission_frequency:
```

Reply:
Should we remove the metric from the argument name and only provide in the docs description as you already do?

Reply:
same applies to other arguments

Reply:
As discussed, since seconds and msec are two different argument fields, the argument name will be kept the same.